base/linalg/bidiag.jl

```diff
@@ -35,7 +35,9 @@ end
 Bidiagonal(A::AbstractMatrix, isupper::Bool)=Bidiagonal(diag(A), diag(A, isupper?1:-1), isupper)
 
 function getindex{T}(A::Bidiagonal{T}, i::Integer, j::Integer)
-    (1<=i<=size(A,2) && 1<=j<=size(A,2)) || throw(BoundsError())
+    if !(1<=i<=size(A,2) && 1<=j<=size(A,2))
+        throw(BoundsError())
```

Comment:
Actually, you'd need to have two separate ones, `BoundsError(A,i)`, or `BoundsError(A,j)`.